react/components/ShippingSimulator/shippingSimulator.css

```diff
@@ -1,11 +1,24 @@
 .shippingContainer {}
 .shippingContainerLoader {}
-.shippingZipcodeLabelLoader {}
-.shippingInputLoader {}
-.shippingZipcodeLabel {}
 .shippingCTA {}
+.shippingInputLoader {}
 .shippingNoMessage {}
 .shippingTable {}
+.shippingTableBody {}
 .shippingTableCell {}
+.shippingTableCellDeliveryEstimate {}
+.shippingTableCellDeliveryName {}
+.shippingTableCellDeliveryPrice {}
+.shippingTableHead
+.shippingTableHead {
+  /* Use the following CSS rule to display correctly */
+  /* display: table-header-group */
```

Comment:
Should this be commented?

Reply:
Yes, it's just to guide the developer if they want to show this HTML element. 

If the developer uses: `display: block;` it will not look correctly.